symplyphysics/laws/hydro/hydrostatic_pressure_from_density_and_depth.py

```diff
@@ -0,0 +1,25 @@
+from sympy import Eq, solve
```

Comment:
There is pending PR with the same law: https://github.com/blackyblack/symplyphysics/pull/130
Since it is not merged yet, I am accepting your PR, but pay attention to existing code next time.